docs/install.md

```diff
@@ -50,13 +50,24 @@ $ docker run --rm --name presidio-api --network mynetwork -d -p 8080:8080 -e WEB
 
 4. Verify that Redis and Traefik/NGINX are installed correctly
 
-5. Deploy from `/charts/presidio`
+5. Verify that Kubernetes has access to the [Docker Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aks).
 
+    - If using a Kubernetes secert, make sure it is registered under 'presidio' namespace
+    
+6. Deploy from `/charts/presidio`
+
+    - If using a public Docker Registry:
     ```sh
     # Based on the DOCKER_REGISTRY and PRESIDIO_LABEL from the previous steps
     $ helm install --name presidio-demo --set registry=${DOCKER_REGISTRY},tag=${PRESIDIO_LABEL} . --namespace presidio
     ```
 
+    - If using a private Docker Registry:
+    ```sh
+    # Based on the DOCKER_REGISTRY and PRESIDIO_LABEL from the previous steps
+    $ helm install --name presidio-demo --set registry=${DOCKER_REGISTRY},tag=${PRESIDIO_LABEL},privateRegistry=${SECRET_NAME} . --namespace presidio
```

Comment:
@itye-msft we should align this file with the README.MD file.
The README.MD file is referring to the scripts your wrote in the deployment dir.

Reply:
what about this @balteravishay  @itye-msft